### PR TITLE
Call deploy_bmh in deploy-edpm

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: docs/source/usage/01_usage.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ are shared among multiple roles:
 - `cifmw_openshift_crio_stats`: (Bool) toggle collecting cri-o stats in CRC deployment.
 - `cifmw_deploy_edpm`: (Bool) toggle deploying EDPM. Default to false.
 - `cifmw_config_nmstate`: (Bool) toggle NMstate networking deployment. Default to false.
+- `cifmw_config_bmh`: (Bool) toggle Metal3 BareMetealHost CRs deployment. Default to false.
```
[the remark this answers is not in the document]
Comment (pablintino) [reply docs/source/usage/01_usage.md:37]: For globals we do not defined a default explicitly, as those vars are not part of a role. A similar behavior can be achieved by combining this two conditions: https://github.com/openstack-k8s-operators/ci-framework/pull/1399/files#diff-5a991b061d05852d376aca719f7a1d3e8d38c5aff30d676f811ce563b504f397R70-R71

**Review by rebtoor — APPROVED** [review]

**Pull request merged**